### DISCUSSION TITLE
Config adjustments

### DISCUSSION THREAD

**Discussion by bantle1984** [emporia-vue-local/esphome#304] [Q&A]
Hi

so I got emporia vue 2 working with esphome and ha

the entity total power I am using in energy dashboard

but it’s not accurate as I do not have enough CT’s 

is there a way to monitor energy with my 2
Main CT’s in instead as a total from the 16 branch circuits?

thanks

**Comment by flaviut** [comment]
There's two questions here I think:

> what do I do about my unused CT clamp slots?

You should delete those lines and all the references to them, otherwise a bunch of gibberish data will be sent into home-assistant. It won't really hurt anything, but it'll take up a bit more disk space and make it harder to find the devices you are using when you're setting up automations.

> How do I get home assistant to not include the device clamps in my total power

this should already be the case. make sure the only device under Grid consumption in the configuration is the Total Daily Energy sensor:

![2024-08-06_22-29](https://github.com/user-attachments/assets/536a2bf9-25e7-4342-a449-e6f4543b8ba4)

**Reply by bantle1984** [reply]
thanks for the help. I may have not worded this great!
i am using the 16 branch circuits. no isssues there.

where does the total daily energy get its readings from? total of all 16 ccts? or from the 2 main feeder ct's? i am hoping its the 2 main feeder ct's!

thanks

**Reply by flaviut** [reply]
Calculated right here from the two main CTs:

```
  - platform: template
    lambda: return id(phase_a_power).state + id(phase_b_power).state;
    update_interval: never   # will be updated after all power sensors update via on_update trigger
    id: total_power
    device_class: power
    state_class: measurement
    unit_of_measurement: "W"
  - platform: total_daily_energy
    name: "Total Daily Energy"
    power_id: total_power
    accuracy_decimals: 0
    restore: false
    filters: *throttle_time
```

**Comment by bantle1984** [comment]
perfect thanks

whats this though? update_interval: never
just curious about this?

as well it seems my voltages are not accurate. one phase 124 and another about 116? any thoughts?

again thanks for all the help very much appreciated

**Reply by flaviut** [reply]
> whats this though? update_interval: never
> just curious about this?

It's quite complicated but there's more details here: https://github.com/emporia-vue-local/esphome/pull/200

> as well it seems my voltages are not accurate. one phase 124 and another about 116? any thoughts?

Take a look at the comments around `calibration: 0.022`

**Comment by bantle1984** [comment]
hey, 

again thanks

had a look and understand now how to get the voltages. only question is 0.022×123.0v = 0.022737
                                                                                                                                  126.5v
that math is wrong from the page i beleaive or am i missing something!lol

**Reply by flaviut** [reply]
You can't just use the other phase's voltage, you need to get the phase voltage from an accurate tool like a multimeter.

**Reply by bantle1984** [reply]
Yeah I have it all sorted out now for the voltage and now accurate… now though I do have circuits ie hot water tank … that should be reading something now and it’s not

I have changed the phases back and forth and still no luck 

any thoughts?
